### PR TITLE
Added default GitHub URL value to omnibus default attributes 

### LINKED ISSUE #2409: Add Documentation for Github Enterprise settings
Documentation required for Github Enterprise feature, make sure to document these at https://github.com/chef/supermarket/blob/main/docs-chef-io/content/supermarket/config_rb_supermarket.md. Without this no one is going to know how to use this new feature.

### PR DESCRIPTION
**Pull request by antima-gupta**
Signed-off-by: antima-gupta <agupta@msystechnologies.com>

### Description
- Added default GitHub URL value.

### Issues Resolved
#2409 

### Check List

- [ ] New functionality includes tests
- [ ] All buildkite tests pass
- [ ] Full omnibus build and tests in buildkite pass
- [x] All commits have been signed-off for the Developer Certificate of Origin. See <https://github.com/chef/chef/blob/main/CONTRIBUTING.md#developer-certification-of-origin-dco>
- [ ] PR title is a worthy inclusion in the CHANGELOG

### CODE REVIEW

**Comment by antima-gupta** [comment]
Buildkite-url: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/245

**Comment by tas50** [comment]
@antima-gupta As part of this can we make sure to document these at https://github.com/chef/supermarket/blob/main/docs-chef-io/content/supermarket/config_rb_supermarket.md. Without this no one is going to know how to use this new feature.

**Comment by antima-gupta** [comment]
> @antima-gupta As part of this can we make sure to document these at https://github.com/chef/supermarket/blob/main/docs-chef-io/content/supermarket/config_rb_supermarket.md. Without this no one is going to know how to use this new feature.

@tas50 Sure, I'll add this in the mentioned document.

**Comment by antima-gupta** [comment]
@tas50 I have added documentation . Please have a look.

**Comment by tas50** [comment]
Thanks @antima-gupta

**Comment by antima-gupta** [comment]
latest build: https://buildkite.com/chef/chef-supermarket-main-omnibus-adhoc/builds/279

File: docs-chef-io/content/supermarket/config_rb_supermarket.md [hunk 1]
```diff
@@ -196,6 +196,43 @@ As a Supermarket feature, GitHub must be enabled via the`default['supermarket'][
 
 : The application client secret that is used to authenticate Supermarket to GitHub. Default value: `nil`.
 
+`default['supermarket']['github_url']`
+
+: The URL of the GitHub that Supermarket connects to. Default value:`https://github.com/`.
```
Comment (IanMadd) [inline docs-chef-io/content/supermarket/config_rb_supermarket.md:201]:
```suggestion
: The GitHub URL that Supermarket connects to. Default value:`https://github.com/`.
```

File: docs-chef-io/content/supermarket/config_rb_supermarket.md [hunk 2]
```diff
@@ -196,6 +196,43 @@ As a Supermarket feature, GitHub must be enabled via the`default['supermarket'][
...
+
+: The application client ID that is used to authenticate Supermarket to GitHub Enterprise. Default value: `nil`.
+
+`default['supermarket']['github_secret']`
+
+: The application client secret that is used to authenticate Supermarket to GitHub Enterprise. Default value: `nil`.
+
+`default['supermarket']['github_enterprise_url']`
+
+: The URL of the GitHub Enterprise that Supermarket connects to. Default value:`nil`.
```
Comment (IanMadd) [inline docs-chef-io/content/supermarket/config_rb_supermarket.md:222]:
```suggestion
: The GitHub Enterprise URL that Supermarket connects to. Default value:`nil`.
```

File: docs-chef-io/content/supermarket/config_rb_supermarket.md [hunk 3]
```diff
@@ -196,6 +196,43 @@ As a Supermarket feature, GitHub must be enabled via the`default['supermarket'][
...
+
+`default['supermarket']['github_client_option_authorize_url']`
+
+: The URL of the GitHub Enterprise Authorize For Example `https://github.YOURDOMAIN.com/login/oauth/authorize`. Default value:`nil`.
```
Comment (IanMadd) [inline docs-chef-io/content/supermarket/config_rb_supermarket.md:226]:
```suggestion
: The URL used to authorize with your implementation of GitHub Enterprise Server. For example: `https://github.example.com/login/oauth/authorize`. Default value: `nil`.
```

File: docs-chef-io/content/supermarket/config_rb_supermarket.md [hunk 4]
```diff
@@ -196,6 +196,43 @@ As a Supermarket feature, GitHub must be enabled via the`default['supermarket'][
...
+
+`default['supermarket']['github_client_option_site']`
+
+: The URL of the GitHub Enterprise Site For Example `https://github.YOURDOMAIN.com/api/v3`. Default value:`nil`.
```
Comment (IanMadd) [inline docs-chef-io/content/supermarket/config_rb_supermarket.md:230]:
```suggestion
: The URL for your implementation of GitHub Enterprise. For example: `https://github.example.com/api/v3`. Default value: `nil`.
```

File: docs-chef-io/content/supermarket/config_rb_supermarket.md [hunk 5]
```diff
@@ -196,6 +196,43 @@ As a Supermarket feature, GitHub must be enabled via the`default['supermarket'][
...
+
+`default['supermarket']['github_client_option_access_token_url']`
+
+: The URL of the GitHub Enterprise Access Token For Example `https://github.YOURDOMAIN.com/login/oauth/access_token`. Default value:`nil`.
```
Comment (IanMadd) [inline docs-chef-io/content/supermarket/config_rb_supermarket.md:234]:
```suggestion
: The URL of the GitHub Enterprise access token: For example: `https://github.example.com/login/oauth/access_token`. Default value:`nil`.
```

**Review by IanMadd — COMMENTED** [review]
Docs suggestions

**Review by tas50 — APPROVED** [review]

**Pull request merged**